dockerfiles/Dockerfile.multiarch

```diff
@@ -0,0 +1,150 @@
+# https://askubuntu.com/a/1369504
+# sudo add-apt-repository ppa:jacob/virtualisation #(for Ubuntu 20.04)
+# sudo apt-get update && sudo apt-get install qemu qemu-user qemu-user-static
+# https://stackoverflow.com/a/60667468
+# docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+# docker buildx rm builder
+# docker buildx create --name builder --use
+# docker buildx inspect --bootstrap
+# docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" -f ./dockerfiles/Dockerfile.multiarch --build-arg FLB_TARBALL=https://github.com/fluent/fluent-bit/archive/v1.8.11.tar.gz ./dockerfiles/
+
+ARG builder-base=debian:bullseye-slim
+ARG production-base=gcr.io/distroless/cc-debian11
+
+FROM busybox:1.34.1 as busybox-include
```

Comment:
has to be this specific?

Reply:
It was before, I just updated it to the latest version.You should pin versions for best practice but we could go with `stable` instead which is the same currently.

Reply:
Otherwise your builds are subject to temporal effects and so not reproducible - what is `stable` changes.